.editorconfig

```diff
@@ -0,0 +1,19 @@
+root = true
+
```

Comment:
No, but I can remove it if you'd like, though it's probably a good idea to leave it. http://editorconfig.org/